frontend/src/metabase/metadata/components/FieldVisibilityPicker/FieldVisibilityPicker.tsx

```diff
@@ -1,46 +1,52 @@
 import { t } from "ttag";
 
 import { FIELD_VISIBILITY_TYPES } from "metabase/lib/core";
-import { Flex, Icon, Select, SelectItem, Text } from "metabase/ui";
-import type Field from "metabase-lib/v1/metadata/Field";
+import {
```

Comment:
This component has been moved and refactored to not use `metabase-lib/v1/metadata/*`